app/assets/stylesheets/users/profile/badges.scss

```diff
@@ -0,0 +1,22 @@
+.profile-section[data-title="badges"] {
+  .badges-row{
+    .donation-badge {
+      font-size: 26px;
+      margin-right: 8%;//** 
+      &:last-child {
+        margin-right: 0px;
+      }
+    }
+    margin-bottom: 10px; //**
```

Comment:
what are these comment stuff for?

Reply:
I think I originally had some comment somewhere and just forgot to get rid of that. I'll take care of it.